plugins/wazo-fanvil/serie-i/plugin-info

```diff
@@ -0,0 +1,119 @@
+{
+    "version": "1.0.0",
+    "description": "Plugin for Fanvil i10X/i16X, i6X series and i51W/i52W/i53W",
+    "description_fr": "Greffon pour les series i10X/i16X, i6X et i51W/i52W/i53W Fanvil",
+    "vendor" : "Fanvil",
+    "vendor.url" : "http://www.fanvil.com",
+    "vendor.description" : "",
+    "vendor.official" : 1,
+    "capabilities": {
+        "Fanvil, i10SV, 2.12.19.9": {
+            "lines": 2,
+            "high_availability": false,
+            "function_keys": 1,
+            "expansion_modules": 0,
+            "switchboard": false,
+            "type": "doorphone",
+            "protocol": "sip"
+        },
+        "Fanvil, i10S, 2.12.19.9": {
+            "lines": 2,
+            "high_availability": false,
+            "function_keys": 1,
+            "expansion_modules": 0,
+            "switchboard": false,
+            "type": "doorphone",
+            "protocol": "sip"
+        },
+        "Fanvil, i10SD, 2.12.19.9": {
+            "lines": 2,
+            "high_availability": false,
+            "function_keys": 2,
+            "expansion_modules": 0,
+            "switchboard": false,
+            "type": "doorphone",
+            "protocol": "sip"
+        },
+        "Fanvil, i16S, 2.12.19.9": {
+            "lines": 2,
+            "high_availability": false,
+            "function_keys": 1,
+            "expansion_modules": 0,
+            "switchboard": false,
+            "type": "doorphone",
+            "protocol": "sip"
+        },
+        "Fanvil, i16SV, 2.12.19.9": {
+            "lines": 2,
+            "high_availability": false,
+            "function_keys": 1,
+            "expansion_modules": 0,
+            "switchboard": false,
+            "type": "doorphone",
```

Comment:
If you want to add phone types, please also add them to the repo `README.md` file.

For now the only documented types we have are:
`type (string): the device type. Can be either deskphone, dect, conference, pager, ata, softphone, generic or gateway`

Please use one of these or add the types you want for documentation uniformity. 
Thank you!